lib/awful/key.lua

```diff
@@ -298,20 +303,44 @@ local function new_common(mod, _keys, press, release, data)
     return setmetatable(ret, obj_mt)
 end
 
-local keygroups = {
+key.keygroups = {
+    -- Left: the keycode in a format which regular awful.key understands.
+    -- Right: the argument of the function ran upon executing the key binding.
     numrow = {},
     arrows = {
         {"Left"  , "Left"  },
         {"Right" , "Right" },
-        {"Top"   , "Top"   },
-        {"Bottom", "Bottom"},
-    }
+        {"Up"    , "Up"    },
```

Comment:
whoops, I should really have added unit tests for this. Can you make a PR just with this?

Reply:
Please check #2999.